documentation/user_docs/docs/manual/Tobyfit.rst

```diff
@@ -62,24 +63,31 @@ difference is that Tobyfit uses instrument information in the sqw objects to
 convolve the |SQW| model(s) with the instrument resolution function using a
 Monte Carlo multi-dimensional integration, and provides some additional methods
 to control how the convolution is carried out.
+A more detailed description of the theory used by TobyFit is given in the
```

Comment:
Are we using a fancy capitalisation scheme for Tobyfit?

Reply:
Why not?

```suggestion
A more detailed description of the theory used by Tobyfit is given in the
```